framework/src/main/java/com/jfcbuilder/builders/XYPolygonBuilder.java

```diff
@@ -35,7 +35,15 @@
 public class XYPolygonBuilder implements IXYAnnotationBuilder<XYPolygonBuilder> {
```

Comment:
Global: review the Javadoc for all classes' `checkBuildPreconditions()` methods and ensure the `@throws` description matches the actual logic.